.github/workflows/lint.yml

```diff
@@ -0,0 +1,12 @@
+name: Lint
+on: [push, pull_request]
+jobs:
+  lint:
+    runs-on: ubuntu-latest
```

Comment:
```suggestion
    runs-on: ubuntu-latest
    if: "!contains(github.event.head_commit.message, '[skip ci]')"
````
Add a way to skip ci

Reply:
May I know specific case where you would want to skip CI?